practice-one/src/assets/css/abstracts/variable.css

```diff
@@ -7,6 +7,7 @@
     --fs-huge: 64px;
     --fs-large: 44px;
     --fs-normal: 36px;
+    --fs-good: 24px;
```

Comment:
You're naming by size. Why is it "good"
 here?

Reply:
> You're naming by size. Why is it "good" here?
I have no idea about new name so that i named it --fs-good, i will fix it